MetaMask/metamask-extension
# fix: GasDetailItem Danger Icon alignment

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24999?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2194

## **Manual testing steps**

1. Go to test dapp
2. Edit code or config so network is busy
3. Click on "Create a token."
4. Observe error icon 

## **Screenshots/Recordings**



### **Before**

![CleanShot 2024-06-03 at 21 12 04@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/44e8fd5f-ad62-4306-908b-9ae615f91794)
![CleanShot 2024-06-03 at 21 16 47@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/865c4a9e-bd3c-419d-b6a3-4dca134ddb55)


### **After**

![CleanShot 2024-06-03 at 21 50 12@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/99b51650-5675-4420-9ebf-c00bc709e106)
![CleanShot 2024-06-03 at 21 59 53@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/5408b8ec-4d9a-4928-8a55-d5bded65e57c)


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
